main.go

```diff
@@ -81,10 +82,11 @@ func readFlags() {
 	}
 	flag.BoolVarP(&flags.version, "version", "V", false, "Print version info and exit")
 	flag.IntVarP(&flags.listenPort, "listen-port", "l", 9000, "The port the Replay Zero proxy will listen on")
-	flag.IntVarP(&flags.defaultTargetPort, "target-port", "t", 8080, "The port the Replay Zero proxy will forward to on localhost")
+	flag.IntVarP(&flags.defaultTargetPort, "target-port", "p", 8080, "The port the Replay Zero proxy will forward to on localhost")
 	flag.BoolVar(&flags.debug, "debug", false, "Set logging to also print debug messages")
 	flag.IntVarP(&flags.batchSize, "batch-size", "b", 1, "Buffer events before writing out to a file")
-	flag.StringVarP(&flags.output, "output", "o", "karate", "Either [karate] or [gatling]")
+	flag.StringVarP(&flags.template, "template", "t", "karate", "Either [karate] or [gatling] or [path/to/custom/template]")
+	flag.StringVarP(&flags.extension, "extension", "e", "", "For custom output template")
```

Comment:
I will update documentation and add details about them after change is finalised

Reply:
See latest review comment for more on docs